hack/kind/kind-config.yaml

```diff
@@ -24,3 +24,7 @@ nodes:
   - containerPort: 30004
     hostPort: 8060
     protocol: TCP
+  # debugger
+  - containerPort: 30005
+    hostPort: 40000
+    protocol: TCP
```

Comment:
not sure about it, but maybe we need also to add the port to firewall rule 
but can you check if it's needed ? ( maybe kind already doing it for us )

Reply:
It worked for me